modules/osm/tags.js

```diff
@@ -255,3 +255,8 @@ export function isColourValid(value) {
     }
     return true;
 }
+
+export var osmTimelessFeatureTagValues = {
+    wood: true, wetland: true, beach: true, cave_entrance: true, peak: true, cliff: true, coastline: true, tree_row: true,
+    water: true, scrub: true, grassland: true, heath: true, bare_rock: true, glacier: true, stream: true, river: true, pond: true, basin: true, lake: true, reservoir: true
```

Comment:
Reservoirs are artificial, and the dates they were built and filled are typically well-documented.

```suggestion
    water: true, scrub: true, grassland: true, heath: true, bare_rock: true, glacier: true, stream: true, river: true, pond: true, basin: true, lake: true
```